config/initializers/stripe.rb

```diff
@@ -1,6 +1,14 @@
-Rails.configuration.stripe = {
-  :publishable_key => ENV['STRIPE_PUBLISHABLE_KEY'],
-  :secret_key      => ENV['STRIPE_API_KEY']
-}
+if Rails.env == 'production'
+  Rails.configuration.stripe = {
+      :publishable_key => ENV['PUBLISHABLE_KEY'],
+      :secret_key => ENV['SECRET_KEY']
+  }
+else
+  Rails.configuration.stripe = {
+      :publishable_key =>"pk_test_NdRxWpMYqwRbDFlOk5CvZcsP",
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Surrounding space missing for operator '=>'.